packages/editor/src/components/post-title/index.js

```diff
@@ -221,15 +218,33 @@ function PostTitle( _, forwardedRef ) {
 				};
 			} );
 		},
-		__unstableDisableFormats: true,
+		__unstableDisableFormats: false,
```

Comment:
This is what allows formats (HTML) to _render_ in the field when in Visual mode.

Reply:
Then this prop can be removed?

Reply:
Yeh it can. Leaving it in for explanation purposes but now it can go.